dist/common/systemd/scylla-helper.slice

```diff
@@ -0,0 +1,27 @@
+[Unit]
+Description=Slice used to run companion programs to Scylla. Memory, CPU and IO restricted
+DefaultDependencies=no
+Before=slices.target
+Wants=-.slice
+After=-.slice
```

Comment:
Are these two needed?

Reply:
It is my understanding that it is, to guarantee that this is initialized after the root slice.

The scylla slice is not top level, so it has to fall within the root slice.

Reply:
It's odd that it would be needed, since it's kind of implied.

Reply:
Man page states that `After` and `Requires` are implicitly added. 
`Wants` is not, but `Wants` is weaker than `Requires`, so I think we can safely delete this.